.concourse/pipeline.yaml.gomplate

```diff
@@ -1985,8 +1981,7 @@ jobs:
       image_resource:
         type: registry-image
         source:
-          repository: thulioassis/bazel-docker-image
-          tag: 2.0.0
+          repository: splatform/catapult
```

Comment:
(See above)

Reply:
Done

Reply:
Hmm, the `repository` part is the same as the merged hash right? Do we need the explicit overrides here?

(As far as I can tell all the other uses of the anchor are fine.)